core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -464,7 +464,7 @@ oppia.directive('conversationSkin', [
               // If the exploration is embedded, use the exploration language
               // as site language. If the exploration language is not supported
               // as site language, English is used as default.
-              var langCodes = $window.GLOBALS.SUPPORTED_SITE_LANGUAGES.map(
+              var langCodes = $window.constants.SUPPORTED_SITE_LANGUAGES.map(
```

Comment:
I wonder why we use $window here and not elsewhere. It's probably OK but did you test manually to verify that it works?